src/features/auth/auth.service.ts

```diff
@@ -11,17 +13,26 @@ export class AuthService {
 
   async validateUser(username: string, pass: string): Promise<any> {
     const user = await this._usersService.findOne(username);
-    if (user && user.password === pass) {
+    if (user && await bcrypt.compare(pass, user.password)) {
       const { password, ...result } = user;
       return result;
     }
     return null;
   }
 
   async login(user: any) {
-    const payload = { username: user.username, sub: user.userId };
-    return {
-      access_token: this._jwtService.sign(payload),
-    };
+    let obj = await this._usersService.getUserStatus(user.username);
+    if (obj.message === 'active') {
```

Comment:
Using hardcoded values `active` is not recommended as the value might change or be modified in the future it will be hard to reflect it. please use  enum or constants